review.md

```diff
@@ -0,0 +1,56 @@
+| "Missing a test for X",                                                                                            | "What is it with you and missing tests?"     |
+| "I've seen this error repeated multiple times, have I mentioned X? try to look into it, it has some nice tools"    | "You keep repeating those types of mistakes" |
+| What is the reason you chose this implementation?                                                                  | huh? Why did you do that?                    |
+
+
+-   Don't forget that in the worst case scenario - you're wrong, and you learned something for the next time.
+
+
+- Remember that the goal isn't solely to suggest changes; it can also serve as an opportunity to ask questions. By asking questions, you can challenge the code owner and assist them in gaining a deeper understanding of their own code. When providing answers, code owners often realize the need for changes they hadn't considered before. Engaging in a discussion will benefit both you and the code owner.   
+  For instance, you can ask questions such as:  
+  - "What is the rationale behind this change?"
+  - "How does this change relate to 'X', which was mentioned in the description?"
+  - "What is the intended purpose of this particular section?"  
+
+  The aim is to foster constructive dialogue that enhances comprehension and cooperation within the team.
+
+
+-   Don't be afraid to give a thumbs down or require changes multiple times! Remember that code reviews are focused on the code itself and should not be taken personally. By separating the code from personal matters, we create an environment that encourages growth and collaboration.
+
+
+-   Try to engage in critical thinking and consider if there are any valuable suggestions you can offer before concluding your review with a simple "LGTM". While there may be instances where no changes are required, it's important to recognize that even a small suggestion can initiate a meaningful conversation.  
+    Here are some aspects to consider during the review process:  
+    - Pay attention to edge cases, such as race conditions, cleanups, and the presence of tests that cover enough use cases without becoming excessive. 
+    - Be on the lookout for missing tests. A good rule of thumb is that if something requires manual execution to see that it works, it's possible that corresponding tests are absent. 
+    - Assess whether the PR focuses on a specific subject. Lengthy or disorganized PRs can make the review process more challenging. It may be beneficial to break them down into smaller, standalone PRs, making it easier to identify errors and shorten the feedback loop. 
+    - Verify that the added functionality does not already exist within the codebase. Take into consideration existing functions, features, or packages that could be utilized instead of reinventing the wheel. 
```

Comment:
You have `code base` and `codebase` on this file. I'd suggest sticking with one of them.

Reply:
done